packages/gitmoji-changelog-core/src/parser.js

```diff
@@ -1,14 +1,28 @@
 const splitLines = require('split-lines')
+const { invert } = require('lodash')
 const groupMapping = require('./groupMapping')
 const emojiMapping = require('./emojiMapping')
 
+const emojiMappingInvert = invert(emojiMapping)
+
 function parseSubject(subject) {
   if (!subject) return {}
 
-  const matches = subject.match(/:(\w*):(.*)/)
-  if (!matches) return { message: subject }
+  let emojiCode
+  let message = subject
 
-  const [, emojiCode, message] = matches
+  const matches = subject.match(/:(\w*):(.*)/)
+  if (matches) {
+    // extract textual emoji
+    [, emojiCode, message] = matches
+  } else {
+    // extract unicode emoji
+    const emoji = subject.substr(0, 1)
+    emojiCode = emojiMappingInvert[emoji]
+    if (emojiCode) {
+      message = subject.substr(1, subject.length)
```

Comment:
Don't you think we should trim the message at the end?
I saw on your test that the commit message is: `<emoji><space><message>`

Maybe this is something that the template take care of?
In this case I think we should move this behaviour in the core module, maybe in an other PR?

Reply:
The message was already trimmed in the return of this function:
https://github.com/frinyvonnick/gitmoji-changelog/blob/64ac5a72fe78877625a8b3cb77183a227a5fb881/packages/gitmoji-changelog-core/src/parser.js#L13-L18